src/custom/state/price/updater.ts

```diff
@@ -132,19 +133,37 @@ export default function FeesUpdater(): null {
   const isUnsupportedTokenGp = useIsUnsupportedTokenGp()
 
   const refetchQuote = useRefetchQuoteCallback()
+  const setQuoteError = useSetQuoteError()
+
   const windowVisible = useIsWindowVisible()
+  const isOnline = useIsOnline()
 
   // Update if any parameter is changing
   useEffect(() => {
     // Don't refetch if window is not visible, or some parameter is missing
     if (!chainId || !sellToken || !buyToken || !typedValue || !windowVisible) return
 
+    // Don't refetch if the amount is missing
     const kind = independentField === Field.INPUT ? 'sell' : 'buy'
     const amount = tryParseAmount(typedValue, (kind === 'sell' ? sellCurrency : buyCurrency) ?? undefined)
-
-    // Don't refetch if the amount is missing
     if (!amount) return
 
+    const quoteParams = { buyToken, chainId, sellToken, kind, amount: amount.raw.toString() }
+
+    // Don't refetch if offline.
+    //  Also, make sure we update the error state
+    if (!isOnline) {
+      if (quoteInfo?.error !== 'offline-browser') {
+        setQuoteError({ ...quoteParams, error: 'offline-browser' })
+      }
+      return
```

Comment:
this return is hard to spot, can you comment why we're returning here

Reply:
https://github.com/gnosis/cowswap/issues/818